<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotsDialog</class>
 <widget class="QDialog" name="PlotsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wykresy</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCustomPlot" name="plot1" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="plot2" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCustomPlot" name="plot3" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="plot4" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
